client/src/components/icons/FavoriteIcon.tsx

```diff
@@ -49,7 +49,7 @@ export const FavoriteIcon = ({itemId}: Props) => {
                 <View style={IconsStyles.iconCircleWrapper}>
                     <View style={IconsStyles.iconCirclePaper}/>
```

Comment:
Add optional backgroundColor props (default: white) and do:
`<View style={IconsStyles.iconCirclePaper}/>` -> `<View style={{...IconsStyles.iconCirclePaper, backgroundColor}}/>`

Remove backgroundColor from iconCirclePaper object so it won't confuse us

Reply:
done